apollo-runtime/src/main/java/com/apollographql/android/impl/ResponseBodyConverter.java

```diff
@@ -53,6 +60,7 @@
         }
       }
       jsonReader.endObject();
+      this.normalizedCache.getCacheStore().mergeOrInsert(resultNormalizer.getRecords());
```

Comment:
Open to discussion on what the right behavior is. I was thinking on an `Exception` we don't normalize anything.